pyproject.toml

```
@@ -58,7 +59,7 @@ dev = ["ruff==0.6.4", # locked so we don't get PRs which fail only due to a lint
 codecarbon = ["codecarbon"]
 speedtask = ["GPUtil>=1.4.0", "psutil>=5.9.8"]
 peft = ["peft>=0.11.0"]
-leaderboard = ["gradio>=5.7.1", "gradio_rangeslider>=0.0.8", "plotly>=5.24.0,<6.0.0"]
+leaderboard = ["gradio>=5.7.1,<6.0.0", "gradio_rangeslider>=0.0.8", "plotly>=5.24.0,<6.0.0"]
```

```suggestion
leaderboard = ["gradio>=5.12.0,<6.0.0", "gradio_rangeslider>=0.0.8", "plotly>=5.24.0,<6.0.0"]
```
You don't want to enforce this to resolve the link clicking?

---

Yeye, I'm doing that rn. Just testing to see if all works.